packages/common/mui/src/lib/card-variants/incentive/incentive.tsx

```diff
@@ -69,37 +89,84 @@ export const Incentive = ({
             )}
             <H4
               textAlign="left"
-              marginBottom={theme.spacing(4)}
-              sx={{ color: formattedTotalRate.includes('-') ? colors.red : '' }}
+              sx={{
+                marginTop: theme.spacing(2),
+                color: formattedTotalRate.includes('-') ? colors.red : '',
+              }}
             >
               {formattedTotalRate}
             </H4>
           </Box>
-
-          <SectionTitle textAlign="left" marginBottom={theme.spacing(1)}>
-            <FormattedMessage defaultMessage={'ORGANIC'} />
-          </SectionTitle>
-          <CardInput
-            textAlign="left"
-            marginBottom={theme.spacing(3)}
-            sx={{ color: formattedRate.includes('-') ? colors.red : '' }}
+          <ContentWrapper
+            hovered={hovered && secondaryIncentiveData ? true : false}
+            theme={theme}
           >
-            {formattedRate}
-          </CardInput>
-          <SectionTitle textAlign="left" marginBottom={theme.spacing(1)}>
-            <PlusIcon width={'9px'} />
-            <FormattedMessage defaultMessage="NOTE INCENTIVE" />
-          </SectionTitle>
-          <CardInput
-            textAlign="left"
-            marginBottom={theme.spacing(3)}
-            sx={{
-              color: formattedIncentiveRate.includes('-') ? colors.red : '',
-            }}
-          >
-            {formattedIncentiveRate}
-          </CardInput>
-        </>
+            <Box>
```

Comment:
@matthew-garrett Is it also possible to put this in its own component and/or maybe merge it with the other hover state? This tsx block is pretty big.

Reply:
There is a lot of refactoring that could be done to these cards but as we are about to replace them is that really worth our time?